platform-includes/configuration/integrations/javascript.browser.mdx

```diff
@@ -0,0 +1,25 @@
+| [`browserApiErrorsIntegration`](./browserapierrors)   |        ✓         |     ✓      |                 |            |                        |
+| [`dedupeIntegration`](./dedupe)                       |        ✓         |     ✓      |                 |            |                        |
+| [`functionToStringIntegration`](./functiontostring)   |        ✓         |            |                 |            |                        |
+| [`globalHandlersIntegration`](./globalhandlers)       |        ✓         |     ✓      |                 |            |                        |
+| [`httpContextIntegration`](./httpcontext)             |        ✓         |            |                 |            |           ✓            |
+| [`inboundFiltersIntegration`](./inboundfilters)       |        ✓         |     ✓      |                 |            |                        |
+| [`linkedErrorsIntegration`](./linkederrors)           |        ✓         |     ✓      |                 |            |                        |
+| [`browserProfilingIntegration`](./browserprofiling)   |                  |            |        ✓        |            |                        |
+| [`browserTracingIntegration`](./browsertracing)       |                  |            |        ✓        |            |           ✓            |
+| [`captureConsoleIntegration`](./captureconsole)       |                  |            |                 |            |           ✓            |
+| [`contextLinesIntegration`](./contextlines)           |                  |     ✓      |                 |            |                        |
+| [`debugIntegration`](./debug)                         |                  |            |                 |            |                        |
+| [`extraErrorDataIntegration`](./extraerrordata)       |                  |            |                 |            |           ✓            |
+| [`httpClientIntegration`](./httpclient)               |                  |     ✓      |                 |            |                        |
+| [`moduleMetadataIntegration`](./modulemetadata)       |                  |            |                 |            |           ✓            |
+| [`rewriteFramesIntegration`](./rewriteframes)         |                  |     ✓      |                 |            |                        |
+| [`replayIntegration`](./replay)                       |                  |            |                 |     ✓      |           ✓            |
+| [`replayCanvasIntegration`](./replaycanvas)           |                  |            |                 |     ✓      |                        |
+| [`reportingObserverIntegration`](./reportingobserver) |                  |     ✓      |                 |            |                        |
+| [`sessionTimingIntegration`](./sessiontiming)         |                  |            |                 |            |                        |
```

Comment:
this should be marked for Additional context, right?

```suggestion
| [`sessionTimingIntegration`](./sessiontiming)         |                  |            |                 |            |                        | ✓
```

Reply:
9d5bf444a324cce306ede04d393d565bfeefd0e8